tests/checks/integration/test_process.py

```diff
@@ -266,6 +284,17 @@ def test_check(self, mock_process):
                     value=expected_value
                 )
 
+            # these are just here to ensure it passes the coverage report.
```

Comment:
Could we maybe define a list similar to `PROCESS_METRIC` (or even include the new page fault metrics in there - not sure if that could break other tests) so that we can do these asserts in a loop and repeat less code?